doc/guide/scripts/correlation_ex3.py

```diff
@@ -20,7 +20,7 @@
 
 # calculate the correlation function G1 and normalize with n to obtain g1
 G1 = qutip.correlation_2op_1t(H, rho0, taus, c_ops, a.dag(), a)
-g1 = G1 / np.sqrt(n[0] * n)
+g1 = np.array(G1) / np.sqrt(n[0] * np.array(n))
```

Comment:
What made it necessary to convert `G1` and `n` to np.array?

Reply:
This is when we made `expect` as an attribute of `Result` instead of a property.